src/main/kotlin/stringcaclulator/Checker.kt

```diff
@@ -0,0 +1,26 @@
+package stringcaclulator
+
+import java.lang.IllegalArgumentException
+
+object Checker {
+    private val NUMBER_REGEX = "[0-9]+|[0-9]+[.][0-9]+".toRegex()
+    private val OPERATOR_REGEX = "[+]|[-]|[*]|[/]".toRegex()
+
+    fun checkMathExpressionSize(mathExpression: List<String>) {
+        require(mathExpression.size >= 3) { throw IllegalArgumentException("연산식의 구성요소는 최소 3개여야 합니다.") }
+    }
+
+    fun checkNumber(number: String) {
+        val isNumber = NUMBER_REGEX.matches(number)
+        if (!isNumber) {
+            throw IllegalArgumentException("숫자 위치에 잘못된 값이 들어있습니다. 잘못된 숫자: $number")
+        }
+    }
```

Comment:
```suggestion
    fun checkNumber(number: String) {
        if (!NUMBER_REGEX.matches(number)) {
            throw IllegalArgumentException("숫자 위치에 잘못된 값이 들어있습니다. 잘못된 숫자: $number")
        }
    }
```
요건 인라인도 가독성이 나쁘지 않아보여! 
그리구 require 사용을 습관화해보자!